listenbrainz/webserver/templates/navbar.html

```diff
@@ -46,6 +50,16 @@
             </li>
           {% endif %}
         </ul>
+      {% else %}
```

Comment:
I'm not really a fan of this block of HTML being duplicated. Can you re-work the IF clause to reduce the duplication?